UnityProject/Assets/Scripts/Managers/TimedEventsManager.cs

```diff
@@ -10,26 +11,62 @@ namespace Managers
 	/// Manager that handles timed game events that only happen under a specific time of the year/month/week/day
 	/// </summary>
 	/// TODO : Allow admins to create and save events for their sever
-	public class TimedEventsManager : SingletonManager<TimedEventsManager>
+	public class TimedEventsManager : SingletonManager<TimedEventsManager>, IInitialise
 	{
 		[SerializeField] private List<TimedGameEventSO> events;
 		private List<TimedGameEventSO> activeEvents = new List<TimedGameEventSO>();
 
 		public List<TimedGameEventSO> ActiveEvents => activeEvents;
 
+		public InitialisationSystems Subsystem { get; }
```

Comment:
make sure you add this to the start-up list

Reply:
I already did